configure.go

```diff
@@ -0,0 +1,28 @@
+package canonicalheader
+
+import (
+	"errors"
+	"strings"
+)
+
+// stringSet is a set-of-nonempty-strings-valued flag.
+type stringSet []string
+
+func (s *stringSet) String() string {
+	return strings.Join(*s, ",")
+}
+
+func (s *stringSet) Set(flag string) error {
+	list := strings.Split(flag, ",")
+
+	*s = make(stringSet, 0, len(list))
+
+	for _, element := range list {
+		if element == "" {
+			return errors.New("empty string")
```

Comment:
Can't we just skip such an element?

Reply:
And it might be useful to `strings.TrimSpace` elements

Reply:
Done!